edpop_explorer/edpopxshell.py

```diff
@@ -185,6 +186,10 @@ def do_stcn(self, args) -> None:
     def do_stcnpers(self, args) -> None:
         'Short Title Catalogue Netherlands – Persons (authors and other contributors)'
         self._query(STCNPersonsReader, args)
+
+    def do_stcnimpr(self, args) -> None:
```

Comment:
I wondered were the `im` in `impr` came from. Eventually I found a mention of `data.typ:impr` in one of the other code changes. I'm guessing this is an STCN convention, but it might be worth clarifying in a comment.

Reply:
I changed this now to `stcnpersons` and `stcnprinters`. `pers` and `impr` are internally used in these APIs at some places, but since they don't appear elsewhere in the Explorer I removed references to them.